modules/RTC/TraceablePeerConnection.js

```diff
@@ -1495,6 +1543,42 @@ TraceablePeerConnection.prototype.setAudioTransferActive = function(active) {
     return changed;
 };
 
+/**
+ * Takes in a *unified plan* offer and inserts the appropriate
+ * parameters for adding simulcast receive support.
+ * @param {} A session description object (with 'type' and 'sdp' fields)
```

Comment:
I think the current style would be to denote the type in the curly braces, so "Object" would be written here.